nussknacker-dist/src/universal/conf/tables-definition.sql

```diff
@@ -1,17 +1,40 @@
 CREATE TABLE transactions
 (
     client_id STRING,
-    amount INT,
-    datetime DATE
+    amount    INT,
+    datetime  DATE
 ) WITH (
     'connector' = 'datagen'
 );
 
 CREATE TABLE transactions_summary
 (
     client_id STRING,
-    amount INT,
-    datetime DATE
+    amount    INT,
+    datetime  DATE
+) WITH (
+      'connector' = 'datagen'
+);
+
+CREATE TABLE all_types_generated
```

Comment:
added this for manual testing

Reply:
Done :+1: I had to add it in 2 places - also in `packageZipTarball` because it wasn't filtering when the filter was only in `Universal / mappings`, so I extracted the filter as a separate function and reused it